src/ui/components/touchableLink/TouchableLink.tsx

```diff
@@ -54,8 +54,12 @@ export function TouchableLink<T extends ElementType = ElementType>({
 
   const handleNavigation = () => {
     if (navigateTo) {
-      const { screen, params, fromRef } = navigateTo
-      ;(fromRef ? navigateFromRef : navigate)(screen, params)
+      const { screen, params, fromRef, withPush } = navigateTo
+      if (!withPush) {
+        ;(fromRef ? navigateFromRef : navigate)(screen, params)
+      } else {
+        ;(fromRef ? pushFromRef : push)(screen, params)
+      }
```

Comment:
je trouve ça plus facile à lire lorsqu'on évite les négations

```suggestion
      if (withPush) {
        ;(fromRef ? pushFromRef : push)(screen, params)
      } else {
        ;(fromRef ? navigateFromRef : navigate)(screen, params)
      }
```